### DISCUSSION TITLE
Super mario odyssey  : 3D is Broken on last beta's

### DISCUSSION THREAD

**Discussion by belmont31** [skyline-emu/skyline#1257] [General]
supper mario odyssey intro just show black screen and no 3d rendering  on last builds: 1089  1090  1091 1092

**Comment by HalogenGam3r** [comment]
You need to supply a save state to reach to ingame.

**Reply by belmont31** [reply]
i know that ... but even if i use save state game dont go ingame anymore ! the last build working with save state is 1087

can i post link with youtube vid here ? to show some tests ??
